.gitignore

```diff
@@ -131,3 +132,6 @@ database/disk_object
 api_test
 #The following file is now a symbolic link only
 scripts/external_chown_script.py
+
+# Deployed tools
+static/tools/omero-image-selection
```

Comment:
The .ccc and static/tools/omero-image-selection directories scare me a little - the files aren't part of the github repo, yet they are expected to be present before the tool runs.
I would prefer if these were passed to the tool using env variables defined in intel_env.sh  (for e.g. OMERO_IMAGE_SELECTION_VIRTENV=/cluster_share/.../venv. These directories would need to be setup before galaxy runs on all clusters and would be explicit in the intel_env file.